File: app/src/main/java/com/OdiousPanda/thefweather/Model/AQI/Attribution.java
package com.OdiousPanda.thefweather.Model.AQI;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Attribution {

    @SerializedName("url")
    @Expose
    public String url;
    @SerializedName("name")
    @Expose
    public String name;

}

File: app/src/main/java/com/odiousPanda/rainbowKt/model/dataSource/aqi/Attribution.kt
package com.odiousPanda.rainbowKt.model.dataSource.aqi


import com.google.gson.annotations.SerializedName

data class Attribution(
    @SerializedName("logo")
    val logo: String,
    @SerializedName("name")
    val name: String,
    @SerializedName("url")
    val url: String
)